<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Reconfiguration for Power Optimisation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>311</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="powerTab">
     <attribute name="title">
      <string>Power Usage</string>
     </attribute>
     <widget class="QTableWidget" name="powerStat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="memTab">
     <attribute name="title">
      <string>Memory Usage</string>
     </attribute>
     <widget class="QTableWidget" name="memStat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>591</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="cpuTab">
     <attribute name="title">
      <string>CPU Usage</string>
     </attribute>
     <widget class="QTableWidget" name="cpuStat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QRadioButton" name="admin">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin Mode</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="automatic">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Automatic Mode</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="reduceBrightness">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Brightness (in percent)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="brightEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn off Display at</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>271</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="advancedOptionsButton">
       <property name="text">
        <string>Advanced Options</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>271</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="hibernateButton">
       <property name="text">
        <string>Hibernate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="turnOffDisplay">
       <property name="text">
        <string>Turn off display</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeButtonHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hibernateButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>hibernateButtonHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>turnOffDisplay</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>turnOffDisplayHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>admin</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>adminModeHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>automatic</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>automaticModeHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>brightOkHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>closeButtonHandler()</slot>
  <slot>hibernateButtonHandler()</slot>
  <slot>turnOffDisplayHandler()</slot>
  <slot>adminModeHandler()</slot>
  <slot>automaticModeHandler()</slot>
  <slot>brightnessSlideHandler()</slot>
  <slot>brightOkHandler()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
